Resources/views/FormFlow/buttons.html.twig

```diff
@@ -5,7 +5,14 @@
 		Thus, all buttons are defined in reverse order and will be reversed again via CSS.
 		See http://stackoverflow.com/questions/1963245/multiple-submit-buttons-specifying-default-button
 	#}
-
+	{%-
+		set craue_label = {
+			button_finish: craue_formflow_button_label_finish | default('button.finish'),
+			button_next: craue_formflow_button_label_next | default('button.next'),
+			button_back: craue_formflow_button_label_back | default('button.back'),
+			button_reset: craue_formflow_button_label_reset | default('button.reset')
+		}
+	-%}

```

Comment:
...and these two lines onto one each? Similar to the other multiline `set` tag below.